005_team_3_agent_design/005_team_3_agent_design.tex

```diff
@@ -1 +1,129 @@
-\chapter{Team 3 Agent Design}\label{team_3_agent_design}
\ No newline at end of file
+\chapter{Team 3 Agent Design}\label{team_3_agent_design}
+
+\section{The Agent}\label{the_agent}
+The agent is designed to show different combinations of behaviour depending on three variables we have used to define any agent: stubbornness, \cite{chun_2005},  morality, \cite{waller_2015}, and mood. These three variables, defined in a range of 0 to 100 will change as the agent encounters a series of scenarios, with its personality shifting as events unravel.  The value of the different variables will affect how the agent takes decisions and acts, all of which will be explained in the following pages. \par
+%%ADD THE MAIN CODE!!!
+As the main function from which all the functionality of the agent is bases we have the \texttt{func (a *CustomAgent3) Run()}. The agent follows then next steps:
+\begin{enumerate}
+    \item The agent starts by updating its personality variables at the beginning of each day calling the necessary function. 
+    \item If the agent was just ‘born’ it will learn an initial value for the average food consumed. 
+    \item In the case the agent has changed floors the personality variables are changed depending on the characteristics of the change. 
+    \item In the case the platform is not empty the agent will eat, the quantity depending on \texttt{a.takeFoodCalculation()}.
+    \item Finally the agent will read and send any messages it considers.
+\end{enumerate}
+
+\subsection{Agent knowledge}
+Agent 3 will be able to remember facts in order to make decisions. The variables in their memory are:
+\begin{enumerate}
+    \item \texttt{floors []int}: stores the floors the agent has been in before. This information is used on reshuffles to set our mood. 
+    \item \texttt{lastHp int}: stores the last recorded HP value. Currently used to check a new day has started.
+    \item \texttt{friends map[uuid.UUID]float64}: the agent is aware of the people they have met during their time in the tower. We assigned the agents uuid with our affection for them. Affection range is 0-1 (0 = dislike, 1 = like). 
+    \item \texttt{foodLastEaten food.FoodType}: stores food last eaten.
```

Comment:
I wouldn't namespace in a report - int conveys enough information here (as a reader won't always have a codebase)

Reply:
(someone external to this project won't know what food.FoodType means)

Reply:
The text has changed, not applicable anymore.